.github/workflows/coverage.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: .tox
-          key: tox-${{ matrix.os }}-${{ hashFiles('pyproject.toml', 'setup.cfg', 'tox.ini') }}
+          key: tox-${{ hashFiles('pyproject.toml', 'setup.cfg', 'tox.ini') }}
```

Comment:
why remove matrix.os from the cache key?

Reply:
Because, we have no matrix here. We only run this on ubuntu 22